data/pug.txt

```diff
@@ -461,7 +461,7 @@ ship "Pug Arfecta"
 		"hull energy" 5
 		"heat generation" 10
 		"energy capacity" 10000
-		"cloaking fuel" -.2
+		"cloaking fuel" -.22
 		weapon
```

Comment:
is it me or is this line and the 2 below it missing a tab? (which is irrelevant to the PR but I just noticed it)